app-deployment-manager/test/deployment/testdata/wordpress-bad-url/wordpress-application.yaml

```diff
@@ -0,0 +1,16 @@
+---
+specSchema: "Application"
+schemaVersion: "0.1"
+$schema: "https://schema.intel.com/catalog.orchestrator/0.1/schema"
+
+name: wordpress
+version: 24.1.18-bad3-url
+description: WordPress is the world's most popular blogging and content management
```

Comment:
@scottmbaker Do you think description should reflect purpose of this DP?. Same comment applies to other DP as well.